asistente_ladm_col/logic/quality_rules/qr_overlapping_right_of_ways_buildings.py

```diff
@@ -0,0 +1,118 @@
+
+        self._id = QR_IGACR3005
+        self._name = "Los servidumbres de tránsito no deben superponerse con construcciones"
+        self._tags = ["igac", "instituto geográfico agustín codazzi", "polígonos", "servidumbre de tránsito con construcciones", "superposición"]
+        self._models = [LADMNames.SURVEY_MODEL_KEY]
+
+        self._errors = {self._ERROR_01: "Los servidumbres de tránsito no deben superponerse con construcciones"}
+
+        # Optional. Only useful for display purposes.
+        self._field_mapping = dict()  # E.g., {'id_objetos': 'ids_punto_lindero', 'valores': 'conteo'}
+
+    def field_mapping(self, names):
+        return {names.ERR_QUALITY_ERROR_T_OBJECT_IDS_F: QCoreApplication.translate("QualityRules", "Right of ways")}
+
+    def layers_config(self, names):
+        return {
+            QUALITY_RULE_LADM_COL_LAYERS: [names.LC_RIGHT_OF_WAY_T,
+                                           names.LC_BUILDING_T],
+            QUALITY_RULE_ADJUSTED_LAYERS: {
+                names.LC_RIGHT_OF_WAY_T: {
+                    ADJUSTED_INPUT_LAYER: names.LC_RIGHT_OF_WAY_T,
+                    ADJUSTED_REFERENCE_LAYER: names.LC_RIGHT_OF_WAY_T},
+                names.LC_BUILDING_T: {
+                    ADJUSTED_INPUT_LAYER: names.LC_BUILDING_T,
+                    ADJUSTED_REFERENCE_LAYER: names.LC_BUILDING_T}
+            }
+        }
+
+    def _validate(self, db, db_qr, layer_dict, tolerance, **kwargs):
+        self.progress_changed.emit(5)
+
+        right_of_way_layer = self._get_layer(layer_dict, db.names.LC_RIGHT_OF_WAY_T)
+        building_layer = self._get_layer(layer_dict, db.names.LC_BUILDING_T)
+        pre_res, res_obj = self._check_prerrequisite_layers(layer_dict)
+        if not pre_res:
+            return res_obj
+
+        dict_uuid_right_of_way = get_uuid_dict(right_of_way_layer, db.names)
+        dict_uuid_building = get_uuid_dict(building_layer, db.names)
+
+        ids, overlapping = GeometryUtils.get_inner_intersections_between_polygons(self, right_of_way_layer, building_layer)
+
+        self.progress_changed.emit(70)
+
+        error_state = LADMData().get_domain_code_from_value(db_qr, db_qr.names.ERR_ERROR_STATE_D,
+                                                            LADMNames.ERR_ERROR_STATE_D_ERROR_V)
+
+        errors = {'geometries': list(), 'data': list()}
+        for key, polygon in zip(ids, overlapping.asGeometryCollection()):
+
+            errors['geometries'].append(polygon)
+
+            error_data = [  # [obj_uuids, rel_obj_uuids, values, details, state]
+                [dict_uuid_right_of_way.get(key[0])],
+                [dict_uuid_building.get(key[1])],
+                None,
+                self._errors[self._ERROR_01],
+                error_state]
+            errors['data'].append(error_data)
+
+        self._save_errors(db_qr, self._ERROR_01, errors)
+
+        self.progress_changed.emit(100)
+
+        if len(errors['geometries']) > 0:
```

Comment:
let's standardize the return as the following example: https://github.com/SwissTierrasColombia/Asistente-LADM-COL/blob/6ed6d0712303429ffbf61a7a29b2d230cd420f7f/asistente_ladm_col/logic/quality_rules/qr_parcel_department_field_valid_code.py#L86

Reply:
fixed in https://github.com/SwissTierrasColombia/Asistente-LADM-COL/commit/8c3cdd521b8b648d49ea12f26f170f6f6424f6dd